CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
-## main
+## 10.0.0-rc.1 - June 9, 2021
```

Comment:
NIT: I think we should leave the ##main in the changelog for post release changes